content-src/asrouter/components/ModalOverlay/ModalOverlay.jsx

```diff
@@ -1,5 +1,13 @@
 import React from "react";
 
+export const ModalOverlayWrapper = props => (<>
+  <div className={`modalOverlayOuter ${props.active ? "active" : ""}`}>
+    <div className={`modalOverlayInner ${props.active ? "active" : ""} ${props.innerClassName || ""}`}>
```

Comment:
Before these were adjacent instead of nested. The outer seems to be the faded background and inner is the actual content. Now because it's nested, it causes everything to be slightly transparent.

https://github.com/mozilla/activity-stream/blob/a0c1c2aead29d9a3ee0c5039abc22769efe7e884/content-src/asrouter/components/ModalOverlay/_ModalOverlay.scss#L7-L9

Reply:
doh! yeah i guess it really wasn't Inner and Outer before

Reply:
nested:
![Screen Shot 2019-04-29 at 2 30 56 PM](https://user-images.githubusercontent.com/438537/56918270-8c65f000-6a8b-11e9-84a3-208d9a39f0e0.png)


sibling:
![Screen Shot 2019-04-29 at 2 30 37 PM](https://user-images.githubusercontent.com/438537/56918258-8708a580-6a8b-11e9-99a1-08e36e183019.png)

Reply:
I suppose it meant to be more "outside" / "inside" vs "outer" / "inner" 😛